src/helpers/CameraHelper.js

```diff
@@ -18,7 +18,7 @@ const _camera = /*@__PURE__*/ new Camera();
 
 class CameraHelper extends LineSegments {
 
-	constructor( camera ) {
+	constructor( camera, colorFrustum = 0xffaa00, colorCone = 0xff0000, colorUp = 0x00aaff, colorTarget = 0xffffff, colorCross = 0x333333 ) {
```

Comment:
An alternative API would be passing an object with all colors as

```js
{
   frustum: 0xff0000,
   cone: 0xff00ff,
  // ...etc
}
```

But I can't recall objects that also use this pattern, so this felt more appropriate

Reply:
I think that the object is a better approach. And it will also solve the naming problem.